### PR TITLE
[policies] Add Red Flag OS policy.

### PR DESCRIPTION
**Pull request by gogv**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [x] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

**Review by arif-ali — APPROVED** [review]

**Review by arif-ali — APPROVED** [review]

**Comment by gogv** [comment]
When will this merge into the main line?

File: sos/policies/distros/redflag.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# See the LICENSE file in the source distribution for further information.
+
+from sos.policies.distros.redhat import RedHatPolicy, OS_RELEASE
+import os
+
+
+class RedFlagPolicy(RedHatPolicy):
+    distro = "Red Flag Asianux Server Linux"
+    vendor = "Red Flag Software"
+    vendor_urls = [('Distribution Website', 'http://www.redflag-os.com')]
```
Comment (pmoravec) [inline sos/policies/distros/redflag.py:18]: Is there some reliable (https) web page? And ideally with information in English? Relying on translators, it is difficult to browse via the (unsecured, not reliable) web page.

Also, is the OS project alive? Per https://en.wikipedia.org/wiki/Red_Flag_Linux , the company developing the OS closed down their office (but Red Flag Desktop 10 was released 2y ago, per [2] at wikipedia..?)

**Comment by TurboTurtle** [comment]
I'm on the fence on this. There seems to have been a trend developing lately around adding a policy and disappearing.

The idea behind policies is first and foremost to do plugin enablement control, so that we can have multiple forms of plugins where needed. The second is to provide a way to refine user experience; through branded messages and automatic configuration of `--upload` options. It also stands that if a distribution has a policy, there will be at least some level (for some amount of time) of ongoing interaction with the project.

As this is a RHEL clone, there's no difference at all in plugin management - there's no plugins called out that need to behave differently. Further, there's no definition for `--upload`.

I'm personally not a fan of loading up policies for the sole sake of branding when there's no actual change in plugin behavior or management, no integrations for potential consumer support, and no indication on ongoing involvement with the project.

But I'll wait to hear other viewpoints from the maintainer's team.

**Comment by gogv** [comment]
May or may not understand.

**Pull request closed without merging**